main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -288,9 +288,34 @@ HistoryPanel.prototype._showApplyOperationsDialog = function() {
   
   elmts.dialogHeader.html($.i18n('core-project/apply-operation'));
   elmts.or_proj_pasteJson.html($.i18n('core-project/paste-json'));
+
+  elmts.operationJsonButton.on('click', async function() {
+    const input = document.createElement('input');
```

Comment:
I suspect we might be able to avoid this creation of a dom element since we already have one in the dialog, no?